ui-v2/app/styles/base/components/breadcrumbs/skin.scss

```diff
@@ -1,36 +1,19 @@
-%breadcrumbs li {
-  list-style-type: none;
+%crumbs {
+  color: $gray-500;
+  text-decoration: none;
 }
-%breadcrumb::before {
-  line-height: 1;
-  font-size: 0.7em;
-  margin-top: 0.2em;
+%crumbs:hover {
+  color: $cobalt-500;
```

Comment:
I remember one of the designers saying we should never use 'brand colors' for non-brand things, so I think this should be just `$blue-500`.

Reply:
updated to be $blue-500.